Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM ubuntu:latest
+
+RUN apt-get update && \
+   apt-get install -y sudo && \
+   apt-get install -y vim
+
+# Install Python and OpenJDK-8
+RUN apt-get update && \
```

Comment:
Instead of doing this, can we run this file instead? https://github.com/oppia/oppia/blob/develop/scripts/install_prerequisites.sh

Reply:
That way, it's all a single source of truth

Reply:
Not really, because at this stage the Oppia folder is not available yet and these are needed for setting other things like the Chrome installation and other tools that are not included in the script.

Reply:
Okay, in that case, please do reference that file since both files need to change in case we need to change the packages in either.

Reply:
I believe we can replace the first apt-get install line (apt-get install -y curl python-setuptools git python-dev python-pip python-yaml) with the prerequisites script. As long as we mkdir /home/oppia early on, and copy over the install_prerequisites script, we can get that going. The rest of the apt-get statements will need to stay there, however.

Reply:
Putting in an edit soon with a slightly reorganized Dockerfile that calls install_prerequisites.

Reply:
That is fine, my concern is preventing discrepancies in the prerequisite packages. At the very least, reference that `install_prerequisites.sh` script.

Reply:
If we don't actually use install_prerequisites, then we'd need a backlink as well from install_prerequisites to Dockerfile, right?

So maybe just including it is better, if possible.

Reply:
Done. Added to Dockerfile.